pages/understanding-json-schema/reference/type.md

```diff
@@ -175,3 +175,64 @@ types. For example, numeric types have a way of specifying a numeric
 range, that would not be applicable to other types. In this reference,
 these validation keywords are described along with each of their
 corresponding types in the following chapters.
+
+## Format[#format]
+
+The `format` keyword conveys semantic information for values that may be difficult or impossible to describe using JSON Schema. Typically, this semantic information is described by other documents. The JSON Schema Validation specification defines several formats, but this keyword also allows schema authors to define their own formats.
+
+For example, because JSON doesn't have a "DateTime" type, dates need to be encoded as strings. `format` allows the schema author to indicate that the string value should be interpreted as a date. By default, `format` is just an annotation and does not affect validation.
+
+Optionally, validator [implementations](../../learn/glossary#implementation) can provide a configuration option to enable `format` to function as an assertion rather than just an annotation. That means that validation fails when, for example, a value with a `date` format isn't in a form that can be parsed as a date. This allows values to be constrained beyond what other tools in JSON Schema, including [Regular Expressions](../../understanding-json-schema/reference/regular_expressions), can do.
+
+> Implementations may provide validation for only a subset of the built-in formats or do partial validation for a given format. For example, some implementations may consider a string an email if it contains an `@`, while others might perform additional checks for other aspects of a well-formed email address.
+
+The JSON Schema specification has a bias toward networking-related formats, likely due to its roots in web technologies. However, custom formats may also be used if the parties exchanging the JSON documents share information about the custom format types. A JSON Schema validator will ignore any format type it does not understand.
+
+### Built-in Formats
+
+It should be noted that `format` is not limited because it only defines a specific set of valid values for formats. Users may define their own custom keywords to work with any specific data type, such as `integer`, `double`, `float`, etc. Below, we cover the commonly used `string` formats specified in the JSON Schema specification.
```

Comment:
Formats can only apply to JSON data types which doesn't include `integer`, `double`, and `float`. Although any type can be used, the only types that would make any sense to use with `format` are `string` and `number`. I think the important thing this paragraph should be communicating is that although the JSON Schema specification only defines `string` formats, it's allowed to define `number` formats as well.

Here's my suggestion, but it could probably use a little more work.

```suggestion
It should be noted that `format` is not limited to a specific set of valid values or types. Users may define their own custom keywords including ones that work with JSON data types other than `string`, such as `number`. Below, we cover the formats specified in the JSON Schema specification.
```